HELP.md

```diff
@@ -0,0 +1,22 @@
+# Read Me First
```

Comment:
Rename to README.md and delete auto generated content. You can actually copy the content from catalog service and modify it a bit to fit this project (Pay attention that this service does not use Spring Profiles and maven profiles yet, so remove references to setting up the profile)